Source/Public/Build-Bicep.ps1

```diff
@@ -27,15 +39,20 @@ function Build-Bicep {
         [switch]$AsHashtable
     )
 
-    
-
     begin {
         if (-not $Script:ModuleVersionChecked) {
             testModuleVersion
         }
         if ($PSBoundParameters.ContainsKey('OutputDirectory') -and (-not (Test-Path $OutputDirectory))) {
             $null = New-Item $OutputDirectory -Force -ItemType Directory -WhatIf:$WhatIfPreference
         }
+        if ($PSBoundParameters.ContainsKey('OutputPath') -and (-not (Test-Path $OutputPath))) {
+            $null = New-Item (Split-Path -Path $OutputPath) -Force -ItemType Directory -WhatIf:$WhatIfPreference
+        }
+        if ($PSBoundParameters.ContainsKey('OutputPath') -and ((Split-path -path $Path -leaf) -notmatch "\.bicep$")) { 
+            Write-Error 'If -Path and -OutputPath is used, only one .bicep file can be used, e.g. -Path "C:\Output\template.bicep" -OutputPath "C:\Output\newtemplate.json"'
```

Comment:
Can we change this to:
'If -Path and -OutputPath parameter is used, only one .bicep file can be used as input to -Path. E.g. -Path "C:\Output\template.bicep" -OutputPath "C:\Output\newtemplate.json"'